src/index.html

```diff
@@ -103,6 +103,13 @@
                 "La modestie ça ne mène nulle part.",
                 "Une question fermée demande une réponse par oui ou par non",
                 "C'est ça qu'on veut ! Délègue à tes collègues ahha",
+                "Femme de peu de foi !",
+                "Notre plan pour conquérir le monde est en marche",
```

Comment:
Il manque un '.' à ta phrase
```suggestion
                "Notre plan pour conquérir le monde est en marche.",
```

Reply:
Y a pas besoin, il sont ajoutés automatiquement =)

Reply:
Le piposselin tellement performant

Reply:
Suite à MAJ de Yannick, ils ne sont plus ajoutés automatiquement.